src/main/webapp/admin_dashboard_style.css

```diff
@@ -0,0 +1,192 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
+
+* {
+	margin: 0;
+	padding: 0;
+	box-sizing: border-box;
+}
+html, body {
+	height: 100%;
+}
+body {
+	font: 15px 'Roboto', sans-serif;
+	color: #222;
+}
+a {
+	cursor: pointer;
+}
+ul {
+	list-style: none;
+}
+.wrapper {
+	display: flex;
+	min-height: 100%;
+}
+nav {
+	width: 200px;
+	background: #eee;
+    border-right: 1px solid #222;
```

Comment:
Could you run this file through a CSS formatter? It's nice to see consistent indentation, spaces, etc. There are online tools you can use, Maven plugins, etc.; maybe cloud shell editor can do it?

I think the main issue is that there's a mixture of tabs and spaces; if your tab width is set to 4, you might not notice but it's quite apparent in the GitHub UI. Maybe a search-and-replace of tabs for 4 spaces would fix matters.